lib/connection/statement.js

```diff
@@ -747,7 +747,7 @@ function createOnStatementRequestSuccRow(statement, context) {
         };
         return;
       }
-      if (body.data.resultIds !== undefined && body.data.resultIds.length > 0) {
+      if (body.data.resultIds != null && body.data.resultIds.length > 0) {
```

Comment:
can we leave only `!body.data.resultIds` instead of `body.data.resultIds != null`?

Reply:
I didn't want to change semantics, but in this case it makes sense, since it would be basically equivalent of  `body.data.resultIds != null && body.data.resultIds != 0`

Reply:
Then this doesn't really make sense anymore unless we invert it, because `!body.data.resultIds` would be more of an equivalent of `body.data.resultIds == null`. I'd leave it like it is.